tests/test_losses/test_pixelwise_loss.py

```diff
@@ -182,7 +182,7 @@ def test_disc_gaussian_log_likelihood_loss(self):
         disc_gaussian_loss = DiscretizedGaussianLogLikelihoodLoss(
             loss_weight=weight, data_info=self.data_info, reduction='mean')
```

Comment:
Do we need to test other reduction ways?

Reply:
Yes, `reduction='batchmean'` is tested line 189.